cardano-cli/src/Cardano/CLI/EraBased/Commands/Genesis.hs

```diff
@@ -81,6 +83,20 @@ data GenesisCreateStakedCmdArgs = GenesisCreateStakedCmdArgs
   , mStakePoolRelaySpecFile :: !(Maybe FilePath) -- ^ Relay specification filepath
   } deriving Show
 
+data GenesisCreateTestNetDataCmdArgs = GenesisCreateTestNetDataCmdArgs
```

Comment:
I agree with Jordan below. Maybe it would be worth copying parameters explanation from [CLI options](https://input-output.atlassian.net/wiki/spaces/HR/pages/1011384363/Laptop+Hardware+Policy) or link them from here?

Reply:
Done :+1: